.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
```

Comment:
## Suggestion
can you change to `"trailingComma": "es5",`? is what we use on other projects and also boilerplate
https://prettier.io/docs/en/options.html#trailing-commas